Doxyfile

```diff
@@ -753,7 +753,7 @@ SHOW_NAMESPACES        = YES
 # The FILE_VERSION_FILTER tag can be used to specify a program or script that
 # doxygen should invoke to get the current version for each file (typically from
 # the version control system). Doxygen will invoke the program by executing (via
-# popen()) the command command input-file, where command is the value of the
+# popen()) the command input-file, where command is the value of the
```

Comment:
I can exclude files (especially this one, which I have offered fixes for in way too many repos...)

Reply:
I don't mind fixing it, since you've found it!